api/edge_api/identities/forwarder.py

```diff
@@ -0,0 +1,63 @@
+import json
```

Comment:
@matthewelwell  I am not 100 percent sure about the name of this module, any ideas?

Reply:
I'd always just err on the side of being as verbose as possible so shall we go with `edge_request_forwarder` or `edge_request_repeater` perhaps?